etc/kayobe/kolla/globals.yml

```diff
@@ -75,7 +75,7 @@ keystone_tag: 7.0.1.10
 kibana_tag: 7.0.1.10
 kolla_toolbox_tag: 7.0.1.10
 logstash_tag: 7.0.1.10
-magnum_tag: 7.0.1.10
+magnum_tag: stein
```

Comment:
are we sure RDO have released a new enough version yet?

Reply:
ah, looks like 8.1.0:
http://mirror.centos.org/centos/7/cloud/x86_64/openstack-stein/openstack-magnum-api-8.1.0-1.el7.noarch.rpm